packages/ui-extensions/src/surfaces/admin/components/CustomerSegmentTemplate/CustomerSegmentTemplate.ts

```diff
@@ -11,17 +11,23 @@ export interface CustomerSegmentTemplateProps {
   title: string;
   /* Localized description(s) of the template. */
   description: string | string[];
-  /* Code snippet to render in the template with syntax highlighting. */
+  /* DEPRECATED - Code snippet to render in the template with syntax highlighting. */
```

Comment:
Just until the changes are made in the segmentation templates app and the admin. Since no other parties are using our APIs we don't need to respect the deprecation cycle.

Reply:
I just wanted it to be clear while we make the transition :)